libs/s3/cortxcli_test_lib.py

```diff
@@ -194,17 +194,19 @@ def csm_user_show_s3accounts(self, csm_user=None, passwd=None):
 
         return status, accounts
 
-    def csm_user_create_s3account(self, s3_user, email, s3_passwd, csm_user=None, passwd=None):
+    def csm_user_create_s3account(self, s3_user, email, s3_passwd, **kwargs):
         """
         Create s3 account user using csm user(default with admin role).
 
-        :param csm_user: Name of the csm user.
-        :param passwd: password of the csm user.
+        # :param csm_user: Name of the csm user.
```

Comment:
why to comment the kwargs documentation ?

Reply:
reverted.